Sources/XMTPiOS/XMTPEnvironment.swift

```diff
@@ -9,7 +9,7 @@ import Foundation
 
 /// Contains hosts an `ApiClient` can connect to
 public enum XMTPEnvironment: String, Sendable {
-	case dev = "dev.xmtp.network",
-	     production = "production.xmtp.network",
+	case dev = "grpc.dev.xmtp.network:443",
```

Comment:
Are we sure this gets used? I think the more important place to update is [here](https://github.com/xmtp/xmtp-ios/blob/main/Sources/XMTPiOS/ApiClient.swift#L78)

Reply:
Feels like a bug that we have these strings in two places. But just updated both for now. @nakajima might make sense to consolidate it into one in the future.

Reply:
It would be good to get them in the same place, doesn't need to happen in this, but maybe a scrub of our constants and putting in the same place

Reply:
Ya, we should drop https://github.com/xmtp/xmtp-ios/blob/main/Sources/XMTPiOS/ApiClient.swift#L78, it got introduced during the first round of Rust changes a while ago but isn't necessary.